api/nakadi-event-bus-api.yaml

```diff
@@ -681,16 +564,8 @@ paths:
             description: An array of `Partition`s
             items:
               $ref: '#/definitions/Partition'
-        '404':
-          description: EventType not found
-          schema:
-            $ref: '#/definitions/Problem'
-        '500':
-          description: Server error
-          schema:
-            $ref: '#/definitions/Problem'
-        '503':
-          description: Service (temporarily) unavailable
+        '401':
```

Comment:
I would say that 401 is not worth mentioning. It's clear what it's about and since all endpoints are protected by default, I think it would be fine to omit this status code.